BUILD.md

```diff
@@ -142,8 +136,8 @@ Options can be enabled by specifiying `-DKokkos_ENABLE_X`.
 * Kokkos_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE
     * Whether to enable relocatable device code (RDC) for CUDA
     * BOOL Default: OFF
-* Kokkos_ENABLE_CUDA_UVM
-    * Whether to enable unified virtual memory (UVM) for CUDA
+* Kokkos_ENABLE_CUDA_DEFAULT_UM
+    * Whether to use unified memory (UM) by default for CUDA
```

Comment:
Yes to update the description, no to rename.  I suppose this is related to #2289 
This needs more discussion and belong to a separate PR.

Reply:
Yeah we need to discuss this in the meeting.